source/extensions/upstreams/http/metadata/BUILD

```diff
@@ -0,0 +1,21 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+envoy_cc_library(
```

Comment:
Please use `envoy_cc_extension`
and add `category` and `security_posture`

See: https://github.com/envoyproxy/envoy/blob/main/source/extensions/upstreams/http/BUILD#L11